Монин Константин

Kuvadorian opened the pull request:
[Посмотреть решение](https://hruview.herokuapp.com/Kuvadorian/markup-task-4/master/index.html)

Kuvadorian committed:
Complete

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

msmirnov commented on index.css:
```diff
@@ -0,0 +1,81 @@
+
+.short-info:hover
+{
+    font-family: fantasy;
```
Такой шрифт есть далеко не на всех устройствах. Если хочется необычный шрифт, то его нужно подключить.

msmirnov reviewed (changes requested):
При отображении списком, фотки котиков получаются сжатыми на узком экране и растянутыми на широком экране. Это нужно исправить.

Kuvadorian committed:
Сделал исправления

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

Kuvadorian commented:
🍏

msmirnov commented on index.css:
```diff
@@ -0,0 +1,81 @@
+
+.short-info:hover
+{
+    font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
```
У меня в системе нет такого шрифта и он не отображается. Его здесь нужно либо убрать, либо сделать так, чтобы работало на всех устройствах (например, подключить внешний шрифт).

msmirnov commented:
🍅

Kuvadorian committed:
Отформатировал текст

Попытка подключить внешний шрифт не удалась (постараюсь разобраться почему так), поэтому решил отформатировать дефолтный шрифт

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

msmirnov commented:
Так тоже нормально
